com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/LitDetailGUI.cs

```diff
@@ -20,6 +21,7 @@ public static class Styles
                 "Designates a Normal Map to create the illusion of bumps and dents in the details of this Material's surface.");
 
             public static readonly GUIContent detailAlbedoMapScaleInfo = EditorGUIUtility.TrTextContent("Setting the scaling factor to a value other than 1 results in a less performant shader variant.");
+            public static readonly GUIContent detailAlbedoMapFormatError = EditorGUIUtility.TrTextContent("This texture is not in linear space.");
```

Comment:
I wonder if this is clear enough for our users. I think this might be confusing for some users, perhaps expand it a bit.

I think a good error message (is hard :) ) gives you the error and context (runtime parameters) and then points/guides you towards a likely fix.

1. What is wrong
2. What is the context (how we got there)
3. What is the effect/end result (for example, detail texturing is skipped until fixed).
4. What is the correct/expected state
5. Suggestion how to fix it.

Can we give the current format of the texture in the message? so that you know what the format is when it's wrong and what it should be.

These were just my thoughts on this. No specific implementation requirements.

Reply:
This is infobox, so usually we do not go so deep in details for the sake of UX. I kinda copied this from normal texture info box. Do you have particular suggestion into what it should change for better readability?

Reply:
I also think some users will struggle with connecting linear space to the sRGB tick box. Maybe "This texture is marked as sRGB (Gamma space)"?
I think its ok to be a bit more clear since the "fix now" button is not available.